src/components/pages/NavBar.tsx

```diff
@@ -0,0 +1,31 @@
+import cn from 'classnames';
+import { NavLink } from 'react-router-dom';
+import { FC } from 'react';
+
+export const Navbar: FC = () => {
+  const getLinkClass = ({ isActive }: { isActive: boolean }) =>
+    cn('navbar-item', {
+      'has-background-grey-lighter': isActive,
+    });
+
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink className={getLinkClass} to="/">
+            Home
+          </NavLink>
+
+          <NavLink className={getLinkClass} to="/people">
```

Comment:
The same issue as above is present here as well. The `getLinkClass` function is not being used correctly with the `NavLink` component.